packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -106,6 +117,7 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
       isCashIn,
       provider: provider.name,
     })
+    dispatch(selectProvider(provider.name, provider.icon))
```

Comment:
Perhaps this is a matter of personal preference but I think it may be better to not pass the provider icon to the Redux store.

In general, I think the Redux store should include the bare minimum of information we need to do what we want to do. With that in mind, why can't we have all these logos organized in a `const txFeedIcons` object we have in our config file, similar to what we do with other image URLs? Then, when we have the provider name and we need the corresponding icon, we can just look it up from that object?

Reply:
I explored a solution along this lines, but I felt that it made the whole thing significantly more complex. 

I don't agree with Redux having the minimum amount of information. We should definitely not put very big objects there, but I don't think a few strings make a difference.

If I put only the name (or an enum) there then we have to maintain a mapping to the icon as you said and wherever we use this we need to use that mapping. It adds more code and complexity for little benefit imo.

Reply:
Ok, I trust your opinion!